sample-queries/sample-queries.json

```diff
@@ -1608,16 +1608,16 @@
         "id": "ae8adc92-9f78-4065-b4b9-22572bc98492",
         "category": "Search (beta)",
         "method": "POST",
-        "humanName": "search messages",
-        "requestUrl": "/beta/search/query",
-        "docLink": "https://aka.ms/mssearchapi",
+        "humanName": "Search Messages",
+        "requestUrl": "https://graph.microsoft.com/beta/search/query",
```

Comment:
@lanhongvp the https://graph.microsoft.com bit of the request url is not needed for the request url in samples. we just need from the version bit. Therefore this part will be fine if left as
` "requestUrl": "/beta/search/query",`